### DISCUSSION TITLE
Market research

### DISCUSSION THREAD

**Discussion by med4kat** [Nerds-Who-Code/Mental-Health-Tracker#8] [Q&A]
To make sure we are following the rule of a **user comes first** it would be helpful to see what choices users have atm. Are there volunteers to check some of the apps on the market? 

Please chose the app u would like to try, let us know here which app it is and shortly describe pros and cons.

Feel free to add any other apps to the list. 

We have a list of apps:

- [x] 1. https://mindsanctuary.net/
- [ ] 2. http://goodtohear.co.uk/happiness
- [ ] 3. https://moodistory.com/
- [ ] 4. http://www.moodlytics.com/
- [x] 5. https://en.wikipedia.org/wiki/Daylio_(app)
- [x] 6. https://minddoc.com/de/en/app
- [ ] 7. MoodTrack - https://apps.apple.com/us/app/moodtrack-social-diary/id549251057
- [ ] 8. MoodBuddy - https://apps.apple.com/us/app/moodbuddy/id1548470885
- [ ] 9. MoodBuddy Github (different than #8 I think) - https://github.com/chengjieyun59/Mood-Buddy
- [ ] 10. MyMood: https://www.mymood.app/
- [ ] 11. Moodify: https://apps.apple.com/us/app/moodify-tracker/id1479420076
- [x] 12. Daylio https://daylio.net/

**Comment by xintamosaik** [comment]
I take 1. mindsanctuary.

**Comment by med4kat** [comment]
i take 5 Daylio

**Comment by nab-sa** [comment]
I have 6 and 5, Daylio and MindDoc, installed on my phone.

**Reply by med4kat** [reply]
Could u go for MindDoc then? Or u don't mind to review both of them? If so I'll choose another one.

**Reply by nab-sa** [reply]
I'll keep only MindDoc then.

**Comment by nicolecarman** [comment]
Added numbers 7 through 9 to the list
Edit: added number 10 and 11

**Comment by xintamosaik** [comment]
I can't make an account yet in mindsanctuary. Will try out 2. goodtohear in the meantime

**Reply by xintamosaik** [reply]
Unfortunately I can't trfy 2., because it's IOS only.

**Reply by 1Vera1** [reply]
I will try goodtohear!

**Reply by 1Vera1** [reply]
Changes (as their app is called now) is unavailable at my (IOS) app store :(

**Comment by xintamosaik** [comment]
4. Moodlytics does not work on android. Maybe the whole project is orphaned

**Comment by xintamosaik** [comment]
10. Can't find my mood in android store. Also orhaned?

**Reply by xintamosaik** [reply]
12. Will try daylio. Instralled, works. Report later
